### DISCUSSION TITLE
Is Bromite android blockers all web trackers when browsing ?

### DISCUSSION THREAD

**Discussion by Aplesiespi** [bromite/bromite#1966] [Q&A]
Hi friends ! 👋 i want to ask question about Bromite browser. I want to know that to, Is Bromite blocks all trackers when browsing web sites ? Yes, i know it' blocks ads but, trackers are more dangerius and, uncomfortable thing when they not blocked. 

Please can somone tell me more about this ? 

-Thanks 🤗

**Comment by realmebrand** [comment]
In general, it would be impossible to block all web trackers. Please look at the https://github.com/bromite/bromite#features Yes there is perhaps some overlap in the list of these  https://github.com/bromite/filters and trackers.

What is your aim or purpose? Read: 
https://github.com/bromite/bromite#privacy-limitations

**Reply by Aplesiespi** [reply]
Thanks for reply !  Can you please suggest me an more strict  AD Blocker Filter URL ? I don't know how can find one to use. I hope there is also an Tracker Blocker too inside Filter

**Reply by mpaunovic** [reply]
You can enable efficient system-wide blocking without any third-party software:

https://simpledns.plus/kb/198/how-to-enable-dns-over-tls-dot-in-android

My recommendation is AhaDNS:

https://github.com/AhaDNS/setup-guides/blob/master/Android/Android.md

Here's the privacy policy:

https://ahadns.com/privacy-policy/

HTH
